runtime/near-vm-logic/Cargo.toml

```diff
@@ -25,6 +25,7 @@ sha2.workspace = true
 sha3.workspace = true
 tracing = { workspace = true, optional = true }
 
+near-chain = { path = "../../chain/chain" }
```

Comment:
Is this still required? I was really hoping we could avoid adding this dependency.